src/components/RIVTrafficLightsComponent.js

```diff
@@ -62,7 +62,7 @@ export default function RIVTrafficLightsComponent() {
     setOpen(false);
   };
   const handleTooltipOpen = () => {
-    setOpen(true);
+    setOpen((r) => !r);
```

Comment:
```suggestion
    setOpen((prevValue) => !prevValue);
```
I know I told you to just write `setOpen((r) => !r)`, but it could be changed to be clearer :D